src/creatures/players/player.cpp

```diff
@@ -7068,6 +7077,766 @@ void Player::dismount() {
 	defaultOutfit.lookMount = 0;
 }
 
+// Wings
+
+uint8_t Player::getLastWing() const {
+	const int32_t value = getStorageValue(PSTRG_WING_CURRENTWING);
+	if (value > 0) {
+		return value;
+	}
+	return static_cast<uint8_t>(kv()->get("last-wing")->get<int>());
```

Comment:
std::optional may return nullptr, so you need to validate it before getting **it**

```suggestion
	auto wingOpt = kv()->get("last-wing")
	return static_cast<uint8_t>(wingOpt  ? wingOpt ->get<int>() : 0);
```